lib/paper_trail/has_paper_trail.rb

```diff
@@ -269,6 +270,7 @@ def touch_with_version(name = nil)
 
         attributes.each { |column| write_attribute(column, current_time) }
         save!
+        record_update(true) if self.class.paper_trail_options[:on] == []
```

Comment:
"something in my metadata on the version has changed", then shouldn't you just be updating the version record directly instead of generating a new version?  I'm not sure I'm following you.  `touch_with_version` can be called regardless of what your `on` option is.

This line is here to ensure that in cases where `PaperTrail` is essentially set to not do recordings on changes, users can still manually generate versions via `touch_with_version`.  The following line (`save!(:validate => false)`) should generate versions if you have `options[:on]` set to `nil`, or if `:update` is in the collection.  Hope this helps.